homeassistant/components/philips_js/__init__.py

```diff
@@ -36,18 +41,26 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
         username=entry.data.get(CONF_USERNAME),
         password=entry.data.get(CONF_PASSWORD),
     )
+    options = CONFIG_OPTIONS_SCHEMA(dict(entry.options))
```

Comment:
Why are we applying a voluptuous schema here? If we want to have a default we can use `dict.get`?

Reply:
I want to make sure the schema is valid and apply ALL defaults as per the schema rather than adding them one by one wherever we call .get. (imho we should always have schemas for config/options.)

Reply:
There's only one default so far though. I'd wait until there are more then one option.